src/models/UserSettings.model.ts

```diff
@@ -10,6 +10,14 @@ export const ZodSettingsSchema = z.object({
   }),
 });
 
+export const DEFAULT_SETTINGS: z.infer<typeof ZodSettingsSchema> = {
```

Comment:
Why not just use `SettingsSchema` that is defined below instead of doing a new `z.infer`